spec/requests/proverbs_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ProverbsController, type: :request do
+
+
+  let(:user) { create(:user) }
+  let(:valid_attributes) { {"body" => "This is a proverb", "language" => "en", user: user} }
+  let(:invalid_attributes) { {"body" => " ", "language" => "en"} }
+  let!(:valid_session) { login(user) }
+
+  describe "GET #index" do
+    it "assigns all proverbs as @proverbs" do
+      proverb = Proverb.create! valid_attributes
+      get "/api/v1/proverbs", {}, valid_session
+      expect(assigns(:proverbs)).to eq([proverb])
```

Comment:
Why not check `response body` instead? That will allow us also test what the serializers are doing with the object